replications/internal/queue_management.go

```diff
@@ -126,3 +137,115 @@ func (qm *durableQueueManager) UpdateMaxQueueSize(replicationID platform.ID, max
 
 	return nil
 }
+
+// QueuePath returns the path to the replication queues on disk
+func (qm *durableQueueManager) QueuePath() string {
+	return filepath.Join(qm.enginePath, "replicationq")
+}
+
+// ReplicationQueues returns a map of all current replication stream IDs and their corresponding durable queue structs
+func (qm *durableQueueManager) ReplicationQueues() map[platform.ID]*durablequeue.Queue {
+	return qm.replicationQueues
+}
+
+// StartReplicationQueues updates the durableQueueManager.replicationQueues map, fully removing any partially deleted
+// queues (present on disk, but not tracked in sqlite) and opening all current queues
+func (qm *durableQueueManager) StartReplicationQueues(ctx context.Context, store *sqlite.SqlStore) error {
+	// Get contents of replicationq directory
+	entries, err := os.ReadDir(qm.QueuePath())
+	if err != nil {
+		return err
+	}
+
+	errOccurred := false
+
+	for _, entry := range entries {
+		// Skip over non-relevant entries (must be a dir named with a replication ID)
+		if !entry.IsDir() {
+			continue
+		}
+
+		id, err := platform.IDFromString(entry.Name())
+		if err != nil {
+			continue
+		}
+
+		// Check sqlite to see if current replication stream is tracked there
+		q := sq.Select(
```

Comment:
I would prefer to keep this file SQL-less if possible to keep the separation of concerns clear.

Can we refactor so:
1. The `replications/service.go` queries all ID/max-size entries once at startup
2. The output mapping of ^^^ is passed as an input to `StartReplicationQueues` instead of the `sqlite.SqlStore`